doc/related_projects.rst

```diff
@@ -48,6 +48,9 @@ enhance the functionality of scikit-learn's estimators.
   wrapper around scikit-learn that makes it easy to run machine learning
   experiments with multiple learners and large feature sets.
 
+- `sklearn-deap <https://github.com/rsteca/sklearn-deap>`_ Use evolutionary
```

Comment:
The other possible choices seem - Auto-ML and Model Export for production. Might you have any preference?